src/Altinn.AccessManagement/appsettings.json

```diff
@@ -43,17 +43,23 @@
     "DelegationEventQueueAccountKey": "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==",
     "DelegationEventQueueEndpoint": "http://127.0.0.1:10000/devstoreaccount1/"
   },
-  "SblBridgeSettings": {
+  "ResourceRegistrySettings": {
```

Comment:
The changes to the main appsettings.json should be reverted

Reply:
kvSetting too? Don't we need that for the backend to run?